spec/yaml/swiftnav/sbp/navigation.yaml

```diff
@@ -114,9 +127,16 @@ definitions:
 
  - MSG_POS_LLH:
     id: 0x0201
+    public: True
     short_desc: Geodetic Position
     desc: |
-        Geodetic position solution.
+      This single-point position solution message reports the absolute
+      geodetic coordinates and the status (single point absolute vs
+      RTK) of the position solution. If the rover receiver knows
+      single-point point position of the base station and has an RTK
+      solution, this reports a pseudo-absolute position solution using
+      the base station single-point position solution and the rover's
+      RTK baseline vector.
     fields:
```

Comment:
Dido above.  It's not the base station's psuedo absolute position, but its surveyed position.